crypto/cmp/cmp_local.h

```diff
@@ -93,9 +93,9 @@ struct ossl_cmp_ctx_st {
     /* certificate template */
     EVP_PKEY *newPkey; /* explicit new private/public key for cert enrollment */
     int newPkey_priv; /* flag indicating if newPkey contains private key */
-    X509_NAME *issuer; /* issuer name to used in cert template */
+    const X509_NAME *issuer; /* issuer name to used in cert template */
```

Comment:
I am not sure about this change if it forces you to cast away the const in the free calls above.

Reply:
I believe this is no real problem because the `..._free()` falls discard the contents pointed to by the given argument.
Yet to avoid further delays of this PR due to potentially onging discussion I've removed the `const` of the `X509_NAME *` fields (and consequently the cast to non-const on deallocation).